### ISSUE TITLE
`generate csv`: support Ansible and Helm operators

### ISSUE THREAD

**Issue by estroz** [operator-framework/operator-sdk#2584]
## Feature Request

**Is your feature request related to a problem? Please describe.**

`generate csv` does not _officially_ support Ansible and Helm operator projects. The generator will work, but in a limited capacity.

**Describe the solution you'd like**

Official support for both operator types, and tests to ensure they work as well as for Go operators.

Discussed in #2581 

/kind feature
/priority important-longterm
/help

**Comment by tengqm** [comment]
/remove-lifecycle stale

pls don't go away yet, Eric said help wanted ... :)

**Comment by camilamacedo86** [comment]
With the SDK 1.0 release will be possible to integrate Ansible/Helm with OLM via the makefile targets `make bundle` and ` bundle-build`. Also, users are indeed able to generate the package manifests, which is what is generated via the command `generate csv`, as well by adding the following target in their Makefile :  

```
# Options for "packagemanifests".
ifneq ($(origin CHANNEL), undefined)
PKG_CHANNELS := --channel=$(CHANNEL)
endif
ifeq ($(IS_CHANNEL_DEFAULT), 1)
PKG_IS_DEFAULT_CHANNEL := --default-channel
endif
PKG_MAN_OPTS ?= $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)

# Generate package manifests.
packagemanifests: kustomize
	operator-sdk generate kustomize manifests -q --interactive=false
	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
	$(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS) 
```

In this way, I am closing this one as solved for 1.0. 
c/c @estroz
